exercises/D-logical-operators/exercise2.js

```diff
@@ -3,10 +3,42 @@
   ---------------------------------
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
+
+  let num = 10;
+
+function satisfiesRequirements(num) {
+  if (num > 3 && num < 10) {
+    return true;
+  }
+
+  return false;
+}
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return true         
+  } else {
+    return false
+  }
+
+function isBetween5and10(num) {
+   if (num >=5 && num<=10) {
+     return true;
+   }
```

Comment:
This function needs another return for the time the `num` is not between 5 and 10.
After the if you need to add `return false;`